site/en/guides/build/features/jam-detection.md

```diff
@@ -0,0 +1,431 @@
+# Jam Detection
+
+To provide a configurable mechanism for signal jamming detection on any
+OpenThread node, enable the Jam Detection feature.
+
+This feature is useful for device certifications that require the ability to
+detect signal jamming on a specific channel. It can be configured to meet the
+requirements of each type of certification.
+
+## How it works
+
+Jam Detection monitors the RSSI (received signal strength indicator) of a node
+during specified windows of time to determine whether the channel has been
+jammed.
+
+When Jam Detection is enabled:
+
+1.  The Jam Detection State is set to `false`.
+1.  The node samples the RSSI multiple times over each one second interval.
+1.  If the RSSI over that entire one second interval remains above the
+    configured [RSSI Threshold](#threshold) for every sample, that one second
+    interval is considered jammed.
+1.  If an aggregate number of jammed one second intervals is _more than or_
+    _equal to_ the aggregate number of configured [Busy Period](#busy_period)
+    seconds within the preceding configured [Detection Window](#window) seconds
+    at any point in time, the Jam Detection State at that point in time is set
+    to `true`.
+1.  If an aggregate number of jammed one second intervals is _less than_ the
+    aggregate number of configured [Busy Period](#busy_period) seconds within
+    the preceding configured [Detection Window](#window) seconds at any point in
+    time, the Jam Detection State at that point in time is set to `false`.
+
+### History Bitmap
+
+In the [OpenThread API](#openthread) and [`wpantund` properties](#wpantund), a
+bitmap of the preceding 63 seconds is available for retrieval. This bitmap
+indicates whether the RSSI crossed the configured RSSI Threshold at each of the
+preceding 63 seconds.
+
+For example, you might retrieve the following bitmap:
+
+```
+0xC248068C416E7FF0
+```
+
+Converting to binary produces every instance the RSSI went above the configured
+RSSI Threshold during the preceding 63 seconds:
+
+```
+11000010 01001000 00000110 10001100 01000001 01101110 01111111 11110000
+```
+
+If the Detection Window is set to 16 seconds, and the Busy Period is set to 8
+seconds, the Jam Detection State becomes `true` at 51 seconds, as that is the
+first instance where the RSSI Threshold was exceeded at least 8 entire seconds
+in the preceding 16 seconds. In this example, the Jam Detection State remains
+`true` for the next 13 seconds.
+
+```
+11000010 01001000 00000110 10001100 01000001 01101110 011{11111 11110000}
+                                      [00001 01101110 011] = 8 in 16
+```
+
+This bitmap might be represented by the following graph, if -45 dBm was the
+configured RSSI Threshold:
+
+<figure>
+<a href="/guides/images/ot-jam-detection-graph_2x.png"><img src="/guides/images/ot-jam-detection-graph.png" srcset="/guides/images/ot-jam-detection-graph.png 1x, /guides/images/ot-jam-detection-graph_2x.png 2x" border="0" alt="OT Jam Detection" /></a>
```

Comment:
This image isn't loading, I think we need to change the path to be relative to the file

../../images/ot-jam-detection-graph_2x.png

Double-check that it still appears on the DevSite staging with that changed (it should)

If so, you'll have to update /guides/build/commissioning.md too, there's an image at the beginning there that's not loading.